CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
-- Nothing yet!
+- Brotli support added by Dr. Matthew Swabey
```

Comment:
If we remove the deprecation warning from the app's init function, we can add this to the changelog?

- Remove `FLASK_STATIC_DIGEST_GZIP_FILES` config option
- Add `FLASK_STATIC_DIGEST_COMPRESSION` (defaults to `["gzip"]` and replaces the old gzip config option)